code/datums/explosion.dm

```diff
@@ -60,6 +61,13 @@ GLOBAL_LIST_EMPTY(explosions)
 		light_impact_range = min(GLOB.MAX_EX_LIGHT_RANGE, light_impact_range)
 		flash_range = min(GLOB.MAX_EX_FLASH_RANGE, flash_range)
 		flame_range = min(GLOB.MAX_EX_FLAME_RANGE, flame_range)
+		
+	if(!ignorecap && epicenter.z == ZLEVEL_CITYOFCOGS)
```

Comment:
I mean, it's a shim, but I can't think of a better way at the moment so whatever